packages/react/src/modules/in-app-message/components/Banner/styles.css

```diff
@@ -0,0 +1,54 @@
+.iam-banner {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #ffffff;
+  padding: 16px 24px;
+  border-radius: 8px;
+  border: 0.5px solid #e0e1e6;
+  box-shadow: 0px 12px 20px rgba(28, 32, 36, 0.1);
+  gap: 16px;
+}
+.iam-banner__message {
+  min-width: 380px;
+}
+.iam-banner__title {
+  color: #1c2024;
```

Comment:
It might also be nice to have a default `iam` theme that everything "extends" from defined as CSS vars?

Reply:
Yeah, I like that. Will likely be able to pull a lot of the CSS variables into a shared `iam` scope for all of the ootb components